bin/near

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env node
 
-const gulp = require('gulp');
```

Comment:
Can we still do command line near .... after near -> bin/near?

Reply:
@janedegtiareva do you mean whether it is in `PATH` when installed by `npm`? It's specified by `bin` config in `package.json`, I changed path there as well.